VisualizationBase/src/Scene.cpp

```diff
@@ -126,6 +142,46 @@ void Scene::removeTopLevelItem(Item* item)
 	scheduleUpdate();
 }
 
+
+ViewItem* Scene::currentViewItem()
+{
+	if (!currentViewItem_)
+	{
+		if (viewItems_.size() == 0)
+			addTopLevelItem(new ViewItem(nullptr, "ProjectView"));
```

Comment:
Instead of calling this, here you should call the new dedicated method that you'll write to create a new view